R/backends.R

```diff
@@ -610,3 +616,33 @@ repair_stanfit_names <- function(x) {
 file_refit_options <- function() {
   c("never", "always", "on_change")
 }
+
+.canonicalize_stan_model <- function(stan_file, overwrite_file = TRUE) {
+  if (os_is_windows()) {
+    stanc_cmd <- "bin/stanc.exe"
+  } else {
+    stanc_cmd <- "bin/stanc"
+  }
+  stanc_flags <- c(
+    "--auto-format",
+    "--canonicalize=deprecations,braces,parentheses" 
```

Comment:
`braces,parentheses` is optional. This just adds { } for single line if/for, and braces cleans up any extra braces.